plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
-        "SystemCheckDatabaseHelp": "Matomo requires either the mysqli extension or both the PDO and pdo_mysql extensions.",
+        "SystemCheckDatabaseHelp": "Matomo requires either the MySQLi extension or both the PDO and pdo_mysql extensions.",
         "SystemCheckDatabaseSSL": "Database SSL Connection",
         "SystemCheckDatabaseSSLCipher": "SSL cipher being used",
-        "SystemCheckDatabaseSSLDisabled": "SSL support in your database server is disabled",
-        "SystemCheckDatabaseSSLNo": "Database server is not compiled with SSL support",
-        "SystemCheckDatabaseSSLNotWorking": "%s is set to '1' but SSL connection is not working",
-        "SystemCheckDatabaseSSLOn": "Your database supports SSL connections but SSL connection is not used. Check your database SSL settings in your Matomo config file",
+        "SystemCheckDatabaseSSLDisabled": "SSL support is turned off on your database server",
+        "SystemCheckDatabaseSSLNo": "The database server is not compiled with SSL support",
+        "SystemCheckDatabaseSSLNotWorking": "%s is set to '1', but the SSL connection is not working",
+        "SystemCheckDatabaseSSLOn": "Your database is not using SSL-connections, but supports it. Check the SSL settings of your database in the Matomo config file",
         "SystemCheckDebugBacktraceHelp": "View::factory won't be able to create views for the calling module.",
-        "SystemCheckError": "An error occured - must be fixed before you proceed",
-        "SystemCheckEvalHelp": "Required by HTML QuickForm and Smarty templating system.",
+        "SystemCheckError": "An error occured and must be fixed before proceeding",
+        "SystemCheckEvalHelp": "Required by HTML QuickForm and the Twig templating system.",
         "SystemCheckExtensions": "Other required extensions",
         "SystemCheckFileIntegrity": "File integrity",
-        "SystemCheckFilterHelp": "You need to configure and rebuild PHP with \"filter\" support enabled (don't use --disable-filter).",
+        "SystemCheckFilterHelp": "You need to configure and rebuild PHP with \"filter\" support (don't use --disable-filter).",
         "SystemCheckFunctions": "Required functions",
-        "SystemCheckShellExecHelp": "We recommend you enable this PHP built-in function. %1$sRead this to learn more.%2$s",
-        "SystemCheckGDFreeType": "GD > 2.x + Freetype (graphics)",
-        "SystemCheckGDHelp": "The sparklines (small graphs) and image graphs (in Matomo Mobile app and Email reports) will not work.",
-        "SystemCheckGlobHelp": "This built-in function has been disabled on your host. Matomo will attempt to emulate this function but may encounter further security restrictions. Functionality may be impacted.",
-        "SystemCheckGzcompressHelp": "You need to enable the zlib extension and gzcompress function.",
+        "SystemCheckShellExecHelp": "It is recommended to turn on this PHP built-in function. %1$sRead this to learn more.%2$s",
```

Comment:
Suggest leaving as is for consitstency and clarity

Reply:
"We" something, shouldn't be used anywhere.
There is no benefit to "enabled" here.